oio_rest/requirements.txt

```
@@ -1,6 +1,6 @@
 requests==2.21.0
 Flask==1.0.2
-psycopg2>=2.7
+psycopg2==2.7
```

2.7 is massively out of date; either use 2.7.9 directly or use a compatibility version:
```suggestion
psycopg2~=2.7.0
```
Also; perhaps we should fix the root cause?

---

We should fix the root cause, yeah. This is just a fix until then.